src/api/plans.ts

```diff
@@ -19,6 +20,17 @@ interface PlanSummary {
   concentration_name: string;
 }
 
+interface PlanInfo {
+  course: number;
+  set: number;
+  prereq: number;
+}
+
+interface CourseInfo {
+  course: number;
+  prerequisites: { needed: number; courses: number[] }[];
```

Comment:
What is the purpose of `needed`? Isn't that always going to just be `courses.length`?

Reply:
that was just in the original stubbed out endpoint, I assumed you had a plan for it.